docs/observation/payload.md

```diff
@@ -0,0 +1,43 @@
+# Format of observation payload
+
+Notifications received by OneDrive have a format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+
+```
+{
+  "name": "me--myrepo--master",
+  "params": {
+    "type": "onedrive",
+    "driveId": "...",
+    "owner": "me",
+    "repo": "myrepo",
+    "ref": "master",
+    "mountpoint": {
+      "path": "/ms/",
+      "root": "/myshare"
+    },
+    "changes": [
+      ...
+    ],
+  }
+}
+```
+The `mountpoint` contains the information how paths in the changes array should be translated by replacing a prefix matching `root` with `path`, e.g. for the following change:
+```
+{
+  "id": "...",
+  "relPath": "/myshare/doc.md",
```

Comment:
the path here is not relative to the drive root; it's absolute. so I would call it `path`